aten/src/ATen/native/SpectralOps.cpp

```diff
@@ -924,17 +924,18 @@ Tensor istft(const Tensor& self, const int64_t n_fft, const optional<int64_t> ho
 
   y = y.slice(2, start, end, 1);
   window_envelop = window_envelop.slice(2, start, end, 1);
-  const auto window_envelop_lowest = window_envelop.abs().min().item().toDouble();
-  if (window_envelop_lowest < 1e-11) {
-    std::ostringstream ss;
-    REPR(ss) << "window overlap add min: " << window_envelop_lowest;
-    AT_ERROR(ss.str());
-  }
 
-  y = (y / window_envelop).squeeze(1);  // size: (channel, expected_output_signal_len)
+  // Apply window normalization on non-tiny indices
+  y = at::where(window_envelop.abs().to(kDouble) > 1e-11, y / window_envelop, y);
```

Comment:
While this fixes the NaNs that could happen in the forward pass, I think that backwards might still have NaN.
I wonder if the only solution to fix the backwards would be to manually implement the backwards, by handling this corner case?

Reply:
@peterbell10 Do you have suggestions? I verified and is sure the gradient of the input is ``nan`` when window_envelop has zero values.

Reply:
Would it work to replace the tiny values with ones?
```cpp
window_envelop = at::where(
    window_envelop.abs() > 1e-11, window_envelop, at::ones({}, window_envelop.options());
y = y / window_envelop;
```

Reply:
It works. Thanks for the suggestion!

Reply:
While this does work, it changes the default values during forward -- what was before zero now is one. Is this ok?